bin/templates/scripts/cordova/lib/build.js

```diff
@@ -87,6 +87,13 @@ module.exports.run = function (argv) {
         }
         return Q.nfcall(fs.writeFile, path.join(__dirname, '..', 'build-extras.xcconfig'), extraConfig, 'utf-8');
     }).then(function () {
+        // Ensure platform scripts are executable as the platform could be added on non-OSX environment
+        shell.find(projectPath).filter(function (file) {
```

Comment:
Should we check `cordova` folder only? (projectPath/cordova). My concern is that there could be too many files in other folders like `www` so this could be slow.